lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,37 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://dzer.ru/korotko-o-glavnom/",
```

Comment:
if you want it to be recursive - leave only one url (domain name of your website)